coprocessor/fhevm-engine/fhevm-listener/src/cmd/mod.rs

```diff
@@ -20,13 +20,17 @@ use clap::Parser;
 
 use rustls;
 
+use tokio_util::sync::CancellationToken;
+
 use crate::contracts::{AclContract, TfheContract};
 use crate::database::tfhe_event_propagate::{ChainId, Database};
+use crate::health_ckeck::{HealthCheck, HealthState};
+
 
 #[derive(Parser, Debug, Clone)]
 #[command(version, about, long_about = None)]
 pub struct Args {
-    #[arg(long, default_value = "ws://0.0.0.0:8746")]
+    #[arg(long, default_value = "ws://0.0.0.0:8545")]
```

Comment:
As a side note, maybe this and other arguments shouldn't have default values and be mandatory, e.g. also the database_url, etc.